packages/financial-templates-lib/src/clients/InsuredBridgeL1Client.ts

```diff
@@ -120,7 +123,14 @@ export class InsuredBridgeL1Client {
       ]);
 
       for (const depositRelayedEvent of depositRelayedEvents) {
+        // TODO: Pull this async function call into above Promise.all if possible.
+        const relayDataForDeposit = await this.bridgePools[depositRelayedEvent.returnValues.l1Token].methods
+          .relays(depositRelayedEvent.returnValues.depositHash)
+          .call();
         const relayData = {
+          bridgePoolAddress: depositRelayedEvent.address,
```

Comment:
These are only added for convenience if we want the InsuredBridgePriceFeed to be able to reconstruct the relay ancillary data.